.github/workflows/close-stale.yml

```diff
@@ -0,0 +1,35 @@
+name: Mark stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - codeql/.github/workflows/close-stale.yml
```

Comment:
I think we can drop the `pull_request` trigger and just do testing via `workflow_dispatch` on branches.
```suggestion
```

Reply:
path is incorrect also. I need to have this run once before it appears as an option for `workflow_dispatch`. I'll remove it later.